lib/compenents/article_container.dart

```diff
@@ -0,0 +1,49 @@
+import 'package:flutter/material.dart';
+
+class ArticleContainer extends StatelessWidget {
+  const ArticleContainer({Key? key, this.date, this.title, this.imageUrl})
+      : super(key: key);
+  final String? title;
```

Comment:
create an article class and create an instance and pass this instance as required parameter for this widget to use inside to make it cleaner